docs/01-app/01-getting-started/04-images-and-fonts.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: How to optimize images and fonts
+nav_title: Images and Fonts
+description: Learn how to optimize images and fonts with Next.js
+related:
+  links:
+    - app/api-reference/components/font
+    - app/api-reference/components/image
+---
+
+Next.js comes with automatic image and font optimizations for better performance and user experience. This page will guide you through how to start using them.
+
+## Handling Static Assets
```

Comment:
Would you define Static Assets as a Vercel term/product name? If not then we can lowercase
```suggestion
## Handling static assets
```